src/wp-includes/user.php

```diff
@@ -1974,13 +1974,70 @@ function email_exists( $email ) {
  *
  * @since 2.0.1
  * @since 4.4.0 Empty sanitized usernames are now considered invalid.
+ * @since n.e.x.t New argument $wp_error has been added.
  *
- * @param string $username Username.
- * @return bool Whether username given is valid.
+ * @param string $username Username to validate.
+ * @param bool $wp_error Whether to return a WP_Error if the username is invalid.
+ *
+ * @return bool|WP_Error Whether username given is valid. True is the username is valid, false or WP_Error otherwise.
```

Comment:
```suggestion
 * @param string $username Username to validate.
 * @param bool   $wp_error Whether to return a WP_Error if the username is invalid.
 * @return bool|WP_Error Whether provided username is valid. True if the username is valid, false or WP_Error otherwise.
```